### PR TITLE
PUI-14415: Add preferred contact methods

### PR DESCRIPTION
**Pull request by gretchenziegler**
[PUI-14415](https://vitals.atlassian.net/browse/PUI-14415)

This PR adds functionality to get a member's preferred contact methods using the VCT `preferredContactMethods` endpoint. We used to inject this information into the members.json, but that's unnecessary, since this information only ever needs to be displayed on a member's profile.

🐞🐶

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ rvm:
 branches:
   only:
     - master
+before_script:
```
Comment (wbajzek) [inline .travis.yml]: This might need to be its own PR

Comment (gretchenziegler) [reply .travis.yml]: Okay, I'll take it out for now.
